tests/post/steps/test_logging.py

```diff
@@ -56,6 +73,60 @@ def _check_loki_ready():
     )
 
 
+@given(parsers.parse("we have set up a '{pod_name}' pod"),
+       target_fixture='pod_creation_ts')
+def set_up_logger_pod(host, nodename, k8s_client, utils_image, pod_name):
+    manifest_template = LOGGER_POD_TEMPLATE.read_text(encoding="utf-8")
+    manifest = string.Template(manifest_template).substitute(
+        name=pod_name,
+        image=utils_image,
+    )
+    manifest_path = str(MANIFESTS_PATH / "{}.yaml".format(pod_name))
+    template_path = str(
+        TEMPLATES_PATH / "{}.yaml".format(pod_name)
+    )
+
+    with host.sudo():
+        host.run_test("mkdir -p %s", str(TEMPLATES_PATH))
+
+        write_template = utils.write_string(host, template_path, manifest)
+        assert write_template.rc == 0, (
+            "Failed to create logger Pod manifest template '{}': {}"
+        ).format(template_path, write_template.stderr)
+
+        pod_creation_ts = int(time.time())
+        manage_static_pod = host.run(
+            "salt-call state.single metalk8s.static_pod_managed "
+            'name=%s source=%s',
+            manifest_path,
+            template_path,
+        )
```

Comment:
Why create a static pod and using salt ? Would be better to create the pod using kubectl (or python k8s) (with a label selector if you really want it to be on bootstrap node, but I think we do not really care here)

Reply:
To not rely on kubectl, that's the only point, but I don't find a good reason for that.
I'll change it.